.pre-commit-config.yaml

```diff
@@ -79,6 +79,12 @@ repos:
       # GoSec
       - id: go-sec-repo-mod
 
+  # Nix hooks
```

Comment:
Just a thought, and I'm looking for input on this, but would it make sense to just put this hook under the `Text file hooks` section instead of its own section?

Reply:
I think it's fine in its own, Nix-specific section.  It's definitely a Nix-specific hook.